sql/returning.go

```diff
@@ -50,6 +52,12 @@ func (p *planner) makeReturningHelper(
 		return rh, nil
 	}
 
+	for _, e := range r {
+		if p.aggregateInExpr(e.Expr) {
+			return rh, fmt.Errorf("aggregate functions are not allowed in RETURNING")
```

Comment:
Is this error from postgres? I think we can do better by mentioning the specific illegal expression